counterpartylib/lib/messages/broadcast.py

```diff
@@ -186,16 +190,17 @@ def parse (db, tx, message):
         cursor.close()
         return
 
+    # stop processing if broadcast is invalid for any reason,
```

Comment:
that's the old block that was already there (except it was in the `for`) so it needs to stay in place for parsing older stuff